docs/GuidGuidRecommender.md

```diff
@@ -0,0 +1,144 @@
+
+# Overview
+
+The GUID-GUID coinstallation recommender system generates recommendations
+for add-ons which are similar to a given add-on,
+where similarity is based on the number of profiles
+which have both add-ons installed.
+
+As recommendations are only surfaced
+for a specific set of add-ons meeting certain criteria,
+and depend only on characteristics of the add-ons themselves,
+recommendation sets for each add-on can be (and are) computed in advance.
```

Comment:
Bit long. What about: 

Recommendations are only surfaced based on the input-data, rather than run-time or user data, and therefore can be (and are) computed in advance.